pkg/provider/ibm/provider_test.go

```diff
@@ -1164,7 +1216,9 @@ func TestGetSecretMap(t *testing.T) {
 				t.Errorf("unexpected error: %s, expected: '%s'", err.Error(), v.expectError)
 			}
 			if err == nil && !reflect.DeepEqual(out, v.expectedData) {
-				t.Errorf("unexpected secret data: expected:\n%+v\ngot:\n%+v", v.expectedData, out)
+				exp := bytes.NewBuffer(v.expectedData["credentials"]).String()
+				actual := bytes.NewBuffer(out["credentials"]).String()
+				t.Errorf("unexpected secret data: expected:\n%+v\nexp:%s\ngot:\n%+v\nactual:%s", v.expectedData, exp, out, actual)
```

Comment:
Is it required to specifically extract credentials key?
If not, can this change be reverted?